crm_compassion/models/weekly_revision.py

```diff
@@ -71,6 +75,8 @@ def create(self, vals):
         web_holds = len(holds.filtered(lambda h: h.channel == 'web'))
         ambassador_holds = len(holds.filtered(
             lambda h: h.channel == 'ambassador'))
+        sub_holds = len(holds.filtered(
+            lambda h: h.channel == 'sub'))
```

Comment:
No, the filter should be : `h.type == 'Sub Child Hold'` (the 'sub' channel does not exist)